sdk/dotnet/Iot/ThingGroup.cs

```diff
@@ -179,14 +179,6 @@ public InputMap<string> Tags
             set => _tags = value;
         }
 
-        [Input("tagsAll")]
```

Comment:
Do we need to add this to the changelog?

Reply:
I am going to say I don't believe so - it was never actually something you could set as it was overwritten..